src/app/repo-select/page.tsx

```
@@ -313,90 +224,27 @@ export default function RepoSelectPage() {
-                          alignItems: 'flex-start',
-                          gap: '4px',
-                          padding: '14px 16px',
-                          border: isSelected ? '2px solid #111827' : '1px solid #e5e7eb',
-                          borderRadius: '8px',
-                          background: isSelected ? '#f9fafb' : 'white',
-                          cursor: 'pointer',
-                          textAlign: 'left',
-                          width: '100%',
-                        }}
+                        className={`flex flex-col items-start gap-1 px-4 py-3.5 rounded-lg cursor-pointer text-left w-full ${isSelected ? 'border-2 border-gray-900 bg-gray-50' : 'border border-gray-200 bg-white'}`}
                       >
```

Biome (2.4.6)

[error] Provide an explicit type prop for the button element.

(lint/a11y/useButtonType)